infra/gcp/clusters/projects/artifact-management-sandbox/main.tf

```diff
@@ -0,0 +1,57 @@
+
+/**
+ * Copyright 2021 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ locals {
+     project_id = "artifact-management-sandbox"
+ }
+
+data "google_organization" "org" {
+  domain = "kubernetes.io"
+}
+
+
+resource "google_project" "project" {
+    name = local.project_id
+    project_id = local.project_id
+    org_id = data.google_organization.org.org_id
+}
```

Comment:
Since this is temporary, I don't think we should use the terraform module `gke-project`. Open to suggestions.